.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: ruby
 rvm:
-  - "2.1.6"
```

Comment:
@pdrakeweb 2.1.10 is the latest stable 2.1.x version but if it's a necessity we can put it back.